src/components/cards/challenge/_partials/ReplyToInvitation.tsx

```diff
@@ -16,9 +16,10 @@ interface MultiSelector {
 export interface InvitationProps {
   invite_id: string;
   team_ref: string;
+  ReplyToInvitationTestId?: string
 }
 
-export default function ReplyToInvitation({ invite_id, team_ref }: InvitationProps) {
+export default function ReplyToInvitation({ invite_id, team_ref,ReplyToInvitationTestId = 'reply-to-invitation' }: InvitationProps) {
```

Comment:
Props and variables should be in camel case, not in Pascal case. please replace the `ReplyToInvitationTestId` with `testId`